scripts/math.js

```diff
@@ -0,0 +1,41 @@
+function runArithmetic(array) {
```

Comment:
What does this naming mean?

Reply:
It means, that we are running arithmetic mean. But I deleted 'mean', maked name  shorter.